os/board/artik05x/src/artik05x_boot.c

```diff
@@ -281,6 +294,7 @@ void board_initialize(void)
 	board_audio_initialize();
 	board_sensor_initialize();
 	board_wdt_initialize();
+	board_spi_initialize();
```

Comment:
It's not related to flash or audio exactly, but spi had better initialized before flash & audio

Reply:
Ok I'll fix it